### ISSUE TITLE
The value binding does not work with optgroup on select type multiple

### ISSUE THREAD

**Issue by ilyavf** [canjs/canjs#2369]
When using select type multiple:
- setting viewModel.vv does not affect DOM.
- clicking on select options does not affect viewModel.

[JS Bin](http://jsbin.com/yojovu/edit?js,console,output)

``` html
<select multiple {($value)}="vv">
    <optgroup label="group1">
        <option value=1>1</option>
        <option value=2>2</option>
    </optgroup>
    <optgroup label="group2">
        <option value=3>3</option>
        <option value=4>4</option>
    </optgroup>
</select>
```

**Comment by justinbmeyer** [comment]
Closing for: https://github.com/canjs/can-stache-bindings/issues/49

**Comment by Nithanaroy** [comment]
How is it fixed? I still see this in `"can": "^3.3.1"`

Created an example https://jsbin.com/medifub/edit?html,js,output1

**Comment by justinbmeyer** [comment]
That jsbin isn't using the 'values' property

**Comment by Nithanaroy** [comment]
Thanks Justin. I tried with `$values` as explained at https://canjs.com/doc/can-util/dom/attr/attr.special.values.html, but still don't see the selected value updated `selection` variable. I want to use the select box with single selection with optgroups

Updated JSBin with `$values`, https://jsbin.com/medifub/edit?html,js,output

**Comment by justinbmeyer** [comment]
I misunderstood what this issue was. Re-opened

**Comment by andrejewski** [comment]
The problem was in the `can-util` project and has been fixed in the `v3.3.6` release of that project.

**Comment by justinbmeyer** [comment]
closed by https://github.com/canjs/can-util/pull/233
